packages/calcite-components/src/utils/floating-ui.ts

```diff
@@ -489,8 +494,6 @@ export function connectFloatingUI(
 
     // initial positioning based on https://floating-ui.com/docs/computePosition#initial-layout
     position: component.overlayPositioning,
-    top: "0",
```

Comment:
@jcfranco do you know why this was here?

It was added here: https://github.com/Esri/calcite-design-system/commit/e220686fa366b86dda2647da25144eb45a4a193d#diff-3c419c81de1cc684c4d1d859881782bc87d089da410675b5af3f6c468b95ef13R458

I'd like to remove this to fix the drag and drop positioning issue. #7979

Reply:
Instead, this is only set once an actual position occurs.

Reply:
Looks like that doc says only the `position` needs to be there initially so the top and left can happen later.